dev/com.ibm.ws.security.openidconnect.client/src/com/ibm/ws/security/openidconnect/client/internal/OidcClientImpl.java

```diff
@@ -425,16 +425,33 @@ public ProviderAuthenticationResult authenticate(HttpServletRequest req,
         return oidcClientAuthenticator.authenticate(req, res, oidcClientConfig);
     }
 
+    private boolean requestHasOidcCookie(HttpServletRequest req) {
+        Cookie[] cookies = req.getCookies();
+        for (int i = 0; i < cookies.length; i++) {
```

Comment:
`HttpServletRequest.getCookies()` can return null, which would cause this line to throw an NPE.

Reply:
good catch, fixed